pkg/sypgp/sypgp.go

```diff
@@ -71,28 +72,35 @@ func AskQuestionNoEcho(format string, a ...interface{}) (string, error) {
 	return string(response), nil
 }
 
-// GetTokenFile returns a string describing the path to the stored token file
-func GetTokenFile() string {
+// getSingularityDir returns the directory where the user's singularity
+// configuration and data is located.
+func getSingularityDir() string {
```

Comment:
I believe this could be in a helper/utility package since it is not related to GPG and is ultimately used throughout packages. Maybe in `internal/pkg/util/fs/layout/manager.go`?

Reply:
Added TODOs for this.

Reply:
hmm... getSingularityDir was changed by another commit. This is no longer part of this PR.